src/UserService.Data.Provider.MsSql.Ef/Migrations/20210831231200_AddUsersAvatarsTable.cs

```diff
@@ -0,0 +1,28 @@
+﻿using LT.DigitalOffice.UserService.Models.Db;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+
+namespace LT.DigitalOffice.UserService.Data.Provider.MsSql.Ef.Migrations
+{
+  [DbContext(typeof(UserServiceDbContext))]
+  [Migration("20210831231200_AddUsersAvatarsTable")]
+  class AddUsersAvatarsTable : Migration
+  {
+    protected override void Up(MigrationBuilder migrationBuilder)
```

Comment:
Add Down method?

Reply:
to drop table?
do we need it?

Reply:
It's just considered good coding practice.